src/components/ErrorComponent/ErrorComponent.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import './ErrorComponent.css';
 
 export const ErrorComponent = ({ errorMessage }) => {
   return (
-    <>
-      <h2>{errorMessage}</h2>
+    <div className='error-page'>
```

Comment:
Would this be better as a semantic tag like <section>?

Reply:
Mmm I love section soup